services/supervisord/supervisord.go

```diff
@@ -486,6 +487,23 @@ func (s *Service) UpdateConfiguration(settings *models.Settings) error {
 // once we start generating it. See alertmanager service.
 
 var templates = template.Must(template.New("").Option("missingkey=error").Parse(`
+{{define "dbaas-controller"}}
+[program:dbaas-controller]
+priority = 7
+command = /usr/sbin/dbaas-controller
+user = pmm
+autorestart = {{ .PerconaTestDbaas }}
+autostart = {{ .PerconaTestDbaas }}
+startretries = 10
+startsecs = 1
+stopsignal = INT
```

Comment:
Any particular reason we use SIGINT instead of SIGTERM?

Reply:
No particular reason. If is SIGTERM enough here I can change it.

Reply:
So can we use the same signal as all blocks below for consistency?

Reply:
Sure. I will change it together with priority.